Snakefile

```diff
@@ -228,7 +321,7 @@ rule generate_dot:
     conda:
         config["poppipe_location"] + "/envs/mandrake.yml"
     threads:
-        64
+        4
```

Comment:
Why did you change all the threads to 4? I think snakemake uses up to the threads specified, so if you have only 4 available it won't try and use 64?